Source/VisualRecognitionV3/Models/Class.swift

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright IBM Corporation 2018
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+/** A category within a classifier. */
+public struct Class {
```

Comment:
I guess "Class" (with a capital "C") is okay to use as a name in Swift.  I'm surprised the generator let you do this though.

Reply:
The `WatsonSwiftCodegen` in `develop` actually specifies `Class` as an explicitly reserved word (see [here][1]). I made some [changes][2] to the reserved words in my `swift` branch though, which happened to include removing the capitalized `Class` word.

Swift is smart enough (in this context, anyway) to support `Class`. Although we may want to change it for consistency with other languages. (Maybe your alternate name pull request will end up creating a different name here?)

[1]: https://github.ibm.com/MIL/watson-swagger-codegen/blob/develop/src/main/java/io/swagger/codegen/WatsonSwiftCodegen.java#L111
[2]: https://github.ibm.com/MIL/watson-swagger-codegen/commit/af833886b4a2b26d3bd5986c8c558660cf7fb6a8